README.md

```diff
@@ -1 +1,143 @@
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 # Hello Microverse
+
+**Hello Microverse** is a HTML and CSS project repository with best practices, follow Github flow and a descriptive and easy to understand README file .
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="">HTML</a></li>
+  </ul>
+  <ul>
+    <li><a href="">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Homepage**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://jojo987n.github.io/hello-microverse/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+sh
+npm install
+
+### Setup
+
+Clone this repository to your desired folder:
+sh
+  cd hello-microverse
+  git clone git@github.com:jojo987N/hello-microverse.git
+
+### Install
+
+Install this project with:
+sh
+  cd Hello-microverse
+  npm install
+
+### Usage
+
+To run the project, execute the following command:
+
+### Deployment
+
+You can deploy this project using:
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@jojo987N](https://github.com/jojo987N)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **About Page**
+- [ ] **Services Page**
+- [ ] **Contact us Page**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+Everybody is welcome to suggest changes in Hello Microverse files.
+
+In order to do it, fork this repository, create a new branch and open a Pull Request from your branch
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **[Question_1]**
+
+  - Setup Node js
+
+- **[Question_2]**
+
+  - Setup Linters
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
**[OPTIONAL BUT NECESSARY FOR IMPROVEMENT]** Please consider adding a License file to your repository to make it more professional and in accordance with the project requirement. Currently, as there is no License file this link is resulting in a 404 ERROR.